src/Stratis.Bitcoin.Features.MemoryPool/TxMemPool.cs

```diff
@@ -379,7 +379,7 @@ public bool AddUnchecked(uint256 hash, TxMempoolEntry entry, SetEntries setAnces
 
             if (this.signals != null)
             {
-                this.signals.Publish(new TransactionAddedToMemoryPool(entry.Transaction));
+                this.signals.Publish(new TransactionAddedToMemoryPoolEvent(entry.Transaction));
```

Comment:
This will cause a build error no? The constructor signature has chamged. Pass the mempool size in here.

Reply:
Oh I see that you used the other constructor. Please remove this one and use the new one only.

Reply:
it doesn't create build error. I have created two constructor signatures.